nbdev/export2html.py

```diff
@@ -391,7 +391,9 @@ def notebook2html(fname=None, force_all=False, n_workers=None, cls=HTMLExporter,
     "Convert all notebooks matching `fname` to html files"
     if fname is None:
         files = [f for f in Config().nbs_path.glob('*.ipynb') if not f.name.startswith('_')]
-    else: files = list(p.parent.glob(p.name))
+    else:
+        p = Path(fname)
+        files = list(p.parent.glob(p.name))
```

Comment:
This was the bug.  I forgot to include `p = Path(fname)`.  I realized that I didn't catch this error because I didn't have great tests.  So I added some tests to this notebook.